lib/structures/User.js

```diff
@@ -41,6 +42,9 @@ class User extends Base {
         if(data.discriminator !== undefined) {
             this.discriminator = data.discriminator;
         }
+        if(data.premium_type !== undefined) {
+            this.premiumType = data.premium_type;   
```

Comment:
Can you also add this to the `toJSON` method at the bottom of the file

Reply:
Done!